core/app/models/spree/payment_method.rb

```diff
@@ -73,5 +76,22 @@ def store_credit?
     def available_for_order?(_order)
       true
     end
+
+    def transaction_limits
+      { minimum: transaction_minimum, maximum: transaction_maximum }
+    end
+
+    # Checks if the total of an Order is within the defined Minimum and Maximum Transaction Amount 
+    def within_transaction_limits?(_order)
+      min_valid = transaction_minimum.to_f <= _order.total
```

Comment:
Yeah, i thought the same. i found that actually in the spree core, so i 
adapted it.


On 16.08.2016 17:44, Marta Wyszyńska wrote:
>
> In core/app/models/spree/payment_method.rb 
> <https://github.com/spree/spree/pull/7536#discussion_r74912537>:
>
> > @@ -73,5 +76,22 @@ def store_credit?
> >      def available_for_order?(_order)
> >        true
> >      end
> > +
> > +    def transaction_limits
> > +      { minimum: transaction_minimum, maximum: transaction_maximum }
> > +    end
> > +
> > +    # Checks if the total of an Order is within the defined Minimum and Maximum Transaction Amount
> > +    def within_transaction_limits?(_order)
> > +      min_valid = transaction_minimum.to_f <= _order.total
>
> @krtschmr <https://github.com/krtschmr> if a variable is used it 
> should not be prepended with an underscore, could you change it to 
> |order| in this method? :)
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub 
> <https://github.com/spree/spree/pull/7536/files/70bf961c41dbbf9d7a8a155e5772ff19c278f10d#r74912537>, 
> or mute the thread 
> <https://github.com/notifications/unsubscribe-auth/AMv1up4XjSb4hi3vDpcd_OtokIqc2ycJks5qgZSkgaJpZM4Jkf12>.
>